### ISSUE TITLE
When i run the command in cmd it doesnt loads

### ISSUE THREAD

**Issue by ZorroMundoYT** [Wieku/danser-go#38]
![image](https://user-images.githubusercontent.com/50798713/96511414-acabe080-1224-11eb-9dc7-bae4297054c5.png)
i tried with "run" and in cmd but in cmd doesnt lets me

**Comment by Wieku** [comment]
I don't know, maybe unpack the zip properly?

**Comment by ZorroMundoYT** [comment]
i have unpacked the zip a lot of times but it still gives the same errors

**Comment by Wieku** [comment]
So other possibility is that danser doesn't have access to `assets` directory

**Comment by ZorroMundoYT** [comment]
but if i do the command out of cmd it loads correctly

**Comment by Wieku** [comment]
Can you show me how your path and command look like in both cases? I think I know what's going on

**Comment by ZorroMundoYT** [comment]
out of cmd: C:\Users\megus\Desktop\Danser\danser-0.3.0b.exe -title="MEGALOVANIA" -difficulty="Burn" -tag=2
in cmd: C:\Users\megus\Desktop\Danser\danser-0.3.0b.exe -t="MEGALOVANIA" -d="Easy" -cursors=1 -tag=1 -speed=1 -pitch=1 -mover=flower

**Comment by Wieku** [comment]
So it's a duplicate of #23
cd to danser directory first before starting

**Comment by ZorroMundoYT** [comment]
oh- ill try that

**Comment by ZorroMundoYT** [comment]
nvm i got a new idea and its working-

**Comment by ZorroMundoYT** [comment]
ill tell u was i was doing
![image](https://user-images.githubusercontent.com/50798713/96534771-0547a180-1256-11eb-976a-c572933f2442.png)
so i was coding that, but i tried to run it when u press "Play", but it wasnt working, and i dont know why, so, when i got the idea it copies the catche file (in this case the batch file) and puts it in the selected folder of the selected file and in the name it puts that u have to open it and the date and time and you can change the name of it, and its done :), btw the batch file contains the info to open sucesfully the app
